0003-interledger-protocol/0003-interledger-protocol.md

```diff
@@ -147,61 +145,62 @@ Connectors implement the interledger protocol to forward payments between ledger
 
-Connectors SHOULD include their ILP address in the [`forwardedBy`](#forwardedby) field in the error. Connectors SHOULD NOT modify errors
-in any other way.
+The ILP Packet has two major consumers, who use it differently:
 
-See below for the [ILP Error Format](#ilp-error-format) and [ILP Error Codes](#ilp-error-codes).
+- Connectors use the [ILP Address][] contained in the packet to route the payment.
+- The receiver of the payment uses the packet to identify the packet and which condition to fulfill.
 
-## Specification
+[ILP Address]: ../0015-ilp-addresses/0015-ilp-addresses.md
 
-### ILP Payment Format
 
-Here is a summary of the fields in the ILP payment packet format:
+#### Relation to ILQP Packets
 
-| Field | Type | Short Description |
-|:--|:--|:--|
-| amount | UInt64 | Amount the destination account should receive, denominated in the asset of the destination ledger |
-| account | Address | Address corresponding to the destination account |
-| data | OCTET STRING | Transport layer data attached to the payment |
+The [Interledger Quoting Protocol](../0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md) is another protocol that exists in the Interledger layer of the Interledger protocol suite. (For more information, see [Interledger Architecture](../0001-interledger-architecture/0001-interledger-architecture.md).) ILP Packets share a type space with ILQP Packets, to avoid ambiguity in case the wrong packet is attached to a transfer or other message.
 
-#### amount
+The type value `1` indicates an ILP Packet. The type values `2` through `7` indicate ILQP Packets.
 
-    UInt64 ::= INTEGER (0..18446744073709551615)
+#### Life-Cycle of the ILP Packet
 
-Amount in discrete units of the receiving ledger's asset type. Note that the amount is counted in terms of the smallest indivisible unit on the receiving ledger.
+The exact use of the ILP Packet depends on the transport protocol you use. In [IPR][], the receiver generates and communicates the packet to the sender. In [PSK][], the sender generates the packet according to agreed-upon rules with a pre-shared secret.
+[IPR]: ../0011-interledger-payment-request/0011-interledger-payment-request.md
 
-#### account
+When the sender prepares a transfer to start the payment, the sender attaches the ILP Packet to the transfer, in the memo field if possible. If a ledger does not support attaching the entire ILP Packet to a transfer as a memo, users of that ledger can transmit the ILP Packet using another system, but MUST be able to correlate transfers and ILP Packets. ***Rome's note: Doesn't this also have some sort of trust implications, like the system for correlating transfers must be trusted to the same extent as the ledger because it could be used to steal money if it's malicious?***
```

Comment:
Other parts of the protocol depend on having an authenticated communication channel between parties that are transacting. If you sent the memo over such a channel with a reference to the transfer ID, there wouldn't be a trust issue.